backend/arkham-api/library/Arkham/Location/Cards.hs

```diff
@@ -9417,3 +9418,14 @@ betweenWorlds =
 emptySpace :: CardDef
 emptySpace =
   location "xempty" "EmptySpace" [] NoSymbol [] BeforeTheBlackThrone
+
+rainyLondonStreets :: CardDef
+rainyLondonStreets =
+  location
+    "09510"
```

Comment:
Ideally this would get moved to the correct numerical location, but I can move it afterwards